### ISSUE TITLE
FR: select all checkboxes command

### ISSUE THREAD

**Issue by mwoz123** [ivan-lednev/obsidian-task-archiver#54]
First of all thanks for great plugin. 

One thing I miss hear is ability to select all checkboxes before archiving. 
Is there a chance to add such command to command palette? 

Thanks

**Comment by ivan-lednev** [comment]
Hello, @mwoz123, please describe the workflow in more detail. I don't get what it means to "select all checkboxes".

**Comment by mwoz123** [comment]
plugin should go thought each line and does a change from
`- [ ] `
to 
`- [x] `

**Comment by ivan-lednev** [comment]
This is already possible with simple 'search and replace' built into Obsidian:
![image](https://github.com/ivan-lednev/obsidian-task-archiver/assets/41428836/d80c0550-1238-4a63-8a3e-e901ed5d61bc)

**Comment by mwoz123** [comment]
right, but it's easier to have it in command palette (especially on mobile)...

**Comment by ivan-lednev** [comment]
Take a look at this plugin: https://github.com/jglev/obsidian-apply-patterns-plugin

It's a generic text replacement tool, and it does what you're describing right in the first demo in REAME 😺
